frontend/src/views/Contributors.vue

```diff
@@ -0,0 +1,56 @@
+<template lang="pug">
+  div
+    h1 Contributors
+    h2(v-if="this.errorMessage") {{ errorMessage }}
+    div(v-for="contributor in contributors" class="result")
+      table(style='width:100%')
+        tr
+          th Name
+          th Github
+          th Portfolio
+          th LinkedIn
+        tr
+          td {{ contributor.name }}
+          td 
+            a(:href="contributor.html_url") {{contributor.login}}
+          td 
+            a(:href="contributor.portfolio") {{ contributor.portfolio && 'portfolio' }}
+          td
+            a(:href="contributor.linkedin") {{ contributor.linkedin && 'linkedin' }}
```

Comment:
Use v-if instead. Also, have you seen `frontend/src/components/mb-a.vue` ? It adds a bit of code that makes external links more secure and standardized.

Untested code. 

```suggestion
          td 
            mb-a(:href="contributor.html_url" :v-if="contributor.login") {{contributor.login}}
          td 
            mb-a(:href="contributor.portfolio" :v-if="contributor.portfolio") Portfolio
          td
            mb-a(:href="contributor.linkedin" :v-if="contributor.portfolio") LinkedIn
```

Reply:
Okay.